### ISSUE TITLE
Log file handler on the process email code

### ISSUE THREAD

**Issue by koriaf** [django-helpdesk/django-helpdesk#908]
Description: I'm always frustrated when it creates a physical file instead of just logging things to the console (like most modern container environments are used to work with). The code is https://github.com/django-helpdesk/django-helpdesk/blob/master/helpdesk/email.py#L79

Is there any specific reason to do that? If that functionality is needed, is if fine if somebody implement setting value to not doing that (so existing setups are untouched)? We are having problems with these files on our docker setup - they are pointless there.

**Comment by gwasser** [comment]
We could certainly add an option to disable the log file. Some folks prefer a record of just the email processing log for later audits or whatever.

**Comment by timthelion** [comment]
I'm not sure why this code is setting up that specific handler. Typically Django's logger is highly configurable: https://docs.djangoproject.com/en/3.1/topics/logging/#examples

**Comment by koriaf** [comment]
> I'm not sure why this code is setting up that specific handler. Typically Django's logger is highly configurable: https://docs.djangoproject.com/en/3.1/topics/logging/#examples

yes, my initial thinking is just to get a custom logger `queue_logger = logging.getLogger(queue.name)` and configure it on project-level if needed; but I think it's too late to change that functionality due to backwards compatibility (somebody may rely on these files already). At least without changing the major version number.

**Comment by timthelion** [comment]
Well now is the perfect time to change this, because the latest release is 0.2 and master is 0.3...

**Comment by gwasser** [comment]
Yes we're about to do a major version change to 0.3 so PRs welcome!

**Comment by uhurusurfa** [comment]
Closing as the PR mentioned above provided the desired functioality
